Source/AppAuthTV/OIDTVAuthorizationService.m

```diff
@@ -159,11 +160,20 @@ + (OIDTVAuthorizationCancelBlock)authorizeTVRequest:(OIDTVAuthorizationRequest *
     });
 
     // Creates the token request that will be used to poll the token endpoint.
-    OIDTokenRequest *pollRequest = [TVAuthorizationResponse tokenPollRequest];
+    OIDTVTokenRequest *pollRequest = [TVAuthorizationResponse tokenPollRequest];
 
     // Starting polling interval (may be increased if a slow down message is received).
     __block NSTimeInterval interval = [TVAuthorizationResponse.interval doubleValue];
 
+    // If no interval is set, use default value of 5 as per RFC.
+    // If interval is set to 0, use value of 1 to prevent infinite polling.
+
```

Comment:
nit: Remove empty line. Don't leave space between a comment and the relevant code.